util/cmd/release/main.go

```diff
@@ -109,9 +109,98 @@ func main() {
 	util.Execute(
 		"ghr",
 		"-t="+token,
+		"-n="+version,
+		"-b='"+notes(version)+"'",
 		"-u=googleapis",
 		"-r=gapic-generator-go",
 		"-c="+commitish,
 		version,
 		stagingDir)
 }
+
+// notes collects the commit messages since the previous tag, aggregates
+// dependency updates into one message, and returns formatted release notes.
+func notes(version string) string {
+	previous := util.ExecuteWithOutput(
+		"git",
+		"describe",
+		"--abbrev=0",
+		"--tags",
+		version+"^",
+	)
+	previous = strings.TrimSpace(previous)
+
+	output := util.ExecuteWithOutput(
+		"git",
+		"log",
+		previous+"..HEAD",
+		"--oneline",
+		"--pretty=format:%s",
+	)
+
+	var hasDeps bool
```

Comment:
Could this be broken out into a function and tested so it is easier to understand what the release notes would look like.

Reply:
Good idea. Will do!